deploy/setup_local_envs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This file seems specialized to your development environment -- makes assumptions on paths. Also, not everyone who is working on webapp is also going to be working on ogusa / taxcalc / btax. I think it would be best to leave this out.

Reply:
Ok that's fine with me.  I found it to be useful while I was debugging, but I see your point.